client/checkout/classic/upe-split.js

```diff
@@ -702,11 +702,14 @@ jQuery( function ( $ ) {
 	/**
 	 * Checks if the customer is using a saved payment method.
 	 *
+	 * @param {string} paymentMethodType Stripe payment method type ID.
 	 * @return {boolean} Boolean indicating whether or not a saved payment method is being used.
 	 */
-	function isUsingSavedPaymentMethod() {
+	function isUsingSavedPaymentMethod( paymentMethodType ) {
 		const paymentMethodSelector =
```

Comment:
```suggestion
	function isUsingSavedPaymentMethod() {
	    const paymentMethodType = getSelectedUPEGatewayPaymentMethod();
		const paymentMethodSelector =
```
I would probably move this inside the function, since this value isn't really dependent on something variable that can only be gathered from outside the function--but this isn't really that important.

Reply:
Then again, having it in this current order does remove you having to call `getSelectedUPEGatewayPaymentMethod()` twice in the checkout event, so there's a benefit there. Ah, I'll leave it up to you, haha.

Reply:
Eventually I didn't move `const paymentMethodType = getSelectedUPEGatewayPaymentMethod();` to the method, because of 

> Then again, having it in this current order does remove you having to call getSelectedUPEGatewayPaymentMethod() twice in the checkout event